ext/blake2_ext/blake2-impl.h

```diff
@@ -1,20 +1,23 @@
 /*
```

Comment:
Well, in addition to the gem signing you are already doing ( 👍 ), you could:

* gpg sign your tags in your git repo, github displays and verifies those signatures visibly now.
* You could also commit the gem artifacts to the repo, and run a sha256 sum on them and publish the checksums. People could verify that the binary downloaded from rubygems matches.
* Github provides functionality where you can upload an artifact on the releases page.

I personally do the gnupg signatures on my gem repo commits, and I sign the gem. I publish instructions for people to learn how to install a signed gem (I bet 99.9% of rubygems users have no idea).  Here is an example:  https://github.com/grempe/tss-rb/blob/master/README.md (see the 'Installation Security' sections)

Honestly, I wish Rubygems moved to a model where all gems *must* be signed. Its really not that hard.